acme/acme/test_util.py

```diff
@@ -94,3 +95,11 @@ def skip_unless(condition, reason):  # pragma: no cover
         return lambda cls: cls
     else:
         return lambda cls: None
+
+def broken_on_windows(function):
+    """Decorator to skip temporarily a broken test on Windows."""
+    reason = 'Test is broken and ignored on windows but should be fixed.'
+    return unittest.skipIf(
+        sys.platform == 'win32'
+        and os.environ.get('SKIP_BROKEN_TESTS_ON_WINDOWS', 'true') == 'true',
```

Comment:
It is temporary. The annotation itself is temporary, until I either correct the test or evict it forever on Windows environments. In that extend, default behavior of @broken_on_windows is to skip the test and avoid to break CI, because theses errors are false positives.

The environment variable is here for development purpose. I set it to false locally to find again broken tests and continue to fix them.

Reply:
That's fair. My initial inclination was the default should be false and we should set it to true in our developer environments. With that approach, if someone naively downloads the Certbot package and runs tests on Windows, they won't pass which I think is appropriate because we do not officially support Windows yet.

It's probably not worth trying to set that environment variable everywhere though so this approach is fine.